src/test/java/org/wmn4j/io/musicxml/MusicXmlReaderDomTest.java

```diff
@@ -449,11 +449,23 @@ public void testReadingScoreWithArticulations() {
 	public void testReadingIncorrectXmlFile() {
 		final MusicXmlReader reader = new MusicXmlReaderDom(true);
 		try {
-			final Score score = reader.readScore(Paths.get(MUSICXML_FILE_PATH + "singleCinvalid.xml"));
+			final Score score = reader.readScore(Paths.get(TestHelper.TESTFILE_PATH + MUSICXML_FILE_PATH + "singleCinvalid.xml"));
 			fail("No exception was thrown when trying to read incorrectly formatted XML file");
 		} catch (final IOException e) {
 
 		}
 
 	}
+
+	@Test
+	public void testValidatingCorrectXmlFile() {
+		final MusicXmlReader reader = MusicXmlReader.getReader(true);
```

Comment:
This could call the constructor of `MusicXmlReaderDom` directly because this test is specifically for that implementation.

Reply:
Fixed :+1: